pyproject.toml

```diff
@@ -1,52 +1,227 @@
 [build-system]
-requires = ["setuptools >= 50", "wheel", "setuptools_scm[toml] >= 6"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "python-inspector"
+version = "0.14.0"
+description = "Cariad Python Inspector version"
+authors = [
+    { name="nexB. Inc. and others", email="info@aboutcode.org"},
+    { name="Helio Chissini de Castro", email="helio.chissini.de.castro@cariad.technology" },
+]
+maintainers = [
+    { name="nexB. Inc. and others", email="info@aboutcode.org"},
+    { name ="Helio Chissini de Castro", email="helio.chissini.de.castro@cariad.technology" },
+]
+readme = "README.md"
+license = "Apache-2.0"
+license-files = [
+    "apache-2.0.LICENSE",
+    "NOTICE",
+    "AUTHORS.rst",
+    "CHANGELOG.rst",
+    "CODE_OF_CONDUCT.rst",
+]
+requires-python = ">= 3.10"
+keywords = [
+    "dependencies",
+    "dependency resolution",
+    "dependency",
+    "open source",
+    "package",
+    "pip",
+    "pypi",
+    "python",
+    "requirements",
+    "resolvelib",
+    "resolver",
+    "SBOM",
+    "sca",
+    "scan",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "aiofiles>=24.1.0",
```

Comment:
Why introduce new dependencies in this PR? I cannot see where they would be used. I'd like to see something simple like a basic conversion from setup to pyproject with no other unrelated changes, otherwise this is hard to review

Reply:
This came from the different branch, i removed all unnecessary extra deps